### PR TITLE
chore(trading): add close position button back to console

### LINKED ISSUE #5351: Re-enable close position button in mainnet console
## Tasks
- [ ] Revert: https://github.com/vegaprotocol/frontend-monorepo/pull/5350
- [ ] Update size of close position transaction to align with core by halving the amount we use (see thread)
- [ ] Only enable on testnet

## Additional information

About the original issue:
https://vegaprotocol.slack.com/archives/C9E6XA8GK/p1701076962684339

About re-enabling:
https://vegaprotocol.slack.com/archives/C8WS7M50E/p1701341967722039

### PR DESCRIPTION
**Pull request by MadalinaRaicu**
# Related issues 🔗

Closes #5351 

# Description ℹ️

Re-enable close position button.

### CODE REVIEW

File: libs/orders/src/lib/components/order-list/order-list.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { memo, forwardRef, useMemo, type ForwardedRef } from 'react';
 import {
-  MAXGOINT64,
+  HALFMAXGOINT64,
```
Comment (edd) [inline libs/orders/src/lib/components/order-list/order-list.tsx:3]: **thought (non-blocking)**: I understand *what* this is doing but not _why_. I imagine this is related to a core bug. I wonder if it is worth a comment either here or near line 154 just to explain why this is the limit, for future reference. 

At least git blame will bring someone to this PR and the bug it closes, so you can choose to ignore this

File: libs/positions/src/lib/positions-manager.spec.tsx [hunk 2]
```diff
@@ -31,9 +31,7 @@ jest.mock('@vegaprotocol/data-provider', () => ({
 }));
 
 describe('PositionsManager', () => {
-  // TODO: Close position temporarily disabled in https://github.com/vegaprotocol/frontend-monorepo/pull/5350
-  // eslint-disable-next-line jest/no-disabled-tests
-  it.skip('should close position with max uint64', async () => {
+  it('should close position with max uint64', async () => {
```
[the remark this answers is not in the document]
Comment (MadalinaRaicu) [reply libs/positions/src/lib/positions-manager.spec.tsx]: updated

File: libs/orders/src/lib/components/order-list/order-list.tsx (hunk 1 above)
Comment (MadalinaRaicu) [reply libs/orders/src/lib/components/order-list/order-list.tsx:3]: updated but I am trying to find the core ticket

File: libs/utils/src/lib/constants.ts [hunk 3]
```diff
@@ -1 +1,3 @@
 export const MAXGOINT64 = '9223372036854775807';
+// The fix for the close position functionality needs MaxInt64/2 for the size
```
Comment (MadalinaRaicu) [inline libs/utils/src/lib/constants.ts:2]:
```suggestion
// The fix for the close position functionality needs MaxInt64/2 for the size 
// Issue: https://github.com/vegaprotocol/vega/issues/10177
// Core PR: https://github.com/vegaprotocol/vega/pull/10178
```

**Review by edd — APPROVED** [review]

**Comment by dalebennett1992** [comment]
this looks good to me
image
image

**Review by dalebennett1992 — APPROVED** [review]

**Pull request merged**